docs/migration/automated-migration.mdx

```diff
@@ -1,13 +1,85 @@
 ---
 sidebar_position: 2
 title: Automated Migration
-description: The migration CLI automatically migrates your v2 app.
+description:
+  Discover the new migrate CLI in Front-Commerce v3, designed to simplify and
+  automate the migration process. Effortlessly upgrade your v2 application to
+  v3, streamlining your development workflow.
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## Guide
 
-add codemod instructions here
+After following the
+[Migration Overview](/docs/3.x/migration/migration-overview), you can run the
```

Comment:
I pointed this to the Migration Overview instead of installation guide, as the installation guide is more aimed at new projects.